src/test/scala/org/apache/spark/sql/delta/OptimisticTransactionSuite.scala

```diff
@@ -594,43 +596,343 @@ class OptimisticTransactionSuite extends QueryTest with SharedSparkSession {
     }
   }
 
-  test("no data change: allow data rearrange when new files concurrently added") {
-    withLog(addA_P1 :: addB_P1 :: Nil) { log =>
-      val tx1 = log.startTransaction()
-      tx1.filterFiles()
+  Seq("true", "false").foreach { toggle =>
```

Comment:
nit: why not boolean values?

Reply:
Because I'm using the `toggle` value in `withSQLConf` which requires pairs of `(String, String)`.  
If you think it's preferable, I'll use boolean values and convert them to String where needed.

Reply:
It'd make it more type-safe so preferable. Thank you.